spine-c/include/spine/Animation.h

```diff
@@ -60,6 +60,8 @@ typedef struct spAnimation {
 } spAnimation;
 
 spAnimation* spAnimation_create (const char* name, int timelinesCount);
+spAnimation* spAnimation_createWithTimelines (const char* name, float duration,
```

Comment:
Can we do without this? We'd like to keep the public API as minimal as possible.